frontend_vue/src/components/ModalAddToken.vue

```diff
@@ -0,0 +1,82 @@
+<template>
```

Comment:
very nice component *chefs kiss*. this is the simplicity that our tokens modal on the console is missing

Reply:
yeah this library is really cool and that's how I think modals in general should work.
It's light, it doesn't require anything to mount randomly around the app, it's easy to understand and to follow. 
I wish we could replace the modals we have on canary console... 🥲

Reply:
we totally can replace the modals in the console